custom_components/wattbox/sensor.py

```diff
@@ -57,9 +57,6 @@ def __init__(self, hass: HomeAssistant, name: str, sensor_type: str) -> None:
         self.sensor_type: str = sensor_type
         self._attr_name = f"{name} {SENSOR_TYPES[self.sensor_type]['name']}"
         self._attr_native_unit_of_measurement = SENSOR_TYPES[self.sensor_type]["unit"]
-        self._attr_suggested_unit_of_measurement = SENSOR_TYPES[self.sensor_type][
```

Comment:
Could you explain what the issue with this was a bit more? It has been a while, but I thought the docs said it was best practice to include it.